### PR TITLE
fix build and values file

### PR DESCRIPTION
**Pull request by calvix**
fix build errors, to start proper testing

### CODE REVIEW

File: .abs/main.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+replace-chart-version-with-git: true
+generate-metadata: true
+chart-dir: ./helm/cluster-aws
```
Comment (fiunchinho) [inline .abs/main.yaml:3]:
```suggestion
chart-dir: ./helm/cluster-gcp
```

File: helm/cluster-gcp/Chart.yaml [hunk 2]
```diff
@@ -2,4 +2,4 @@ apiVersion: v2
 name: cluster-gcp
 description: A helm chart for creating Cluster API clusters with the GCP infrastructure provider (CAPG).
 type: application
-version: [[ .Version ]]
+version: "[[ .Version ]]"
```
Comment (fiunchinho) [inline helm/cluster-gcp/Chart.yaml:5]: wasn't this supposed to be removed when using `abs`?

Comment (calvix) [reply helm/cluster-gcp/Chart.yaml:5]: But I guess I can remove it as it is not needed

Comment (fiunchinho) [reply helm/cluster-gcp/Chart.yaml:5]: I would remove it if it's not actually required.

**Pull request merged**